### DISCUSSION TITLE
Installation issue

### DISCUSSION THREAD

**Discussion by Snipey7** [ekutner/home-connect-hass#155] [Community Support]
I try to migrate from the standard Home Connect integration to this Home Connect Alt.
Without luck.

When I arrive at the step of installing the integration in Devices & Services --> integrations I can select the Home Connect Alt integration, but after clicking it it pops up the message "The component is not configured. Follow the documentation" (in Dutch see screenshot)
![afbeelding](https://user-images.githubusercontent.com/122681557/215469416-b163d023-ff38-45ca-a55a-a1cdeaa5ab81.png)
I have read and understood the instructions, and checked for similar issues in this forum but that does not bring me further to a solution.

I followed all steps of the instruction, several times by now.

The old, 'normal' Home Connect integration worked OK before.

In the mean time I have:
- created a new application in HC, with new credentials which I pasted in the HA configuration under "home_connect_alt:",
- removed the credentials of the 'old' integration from the HA configuration,
- removed the old integration,
- double-checked all the inputs in the HC applicaton

Please some hints what may go wrong here...

**Comment by ekutner** [comment]
it looks like you don't have it properly configured in the configuration,yaml file. Make sure it's properly indented and the field names are correct.

**Comment by Snipey7** [comment]
I don't see it. Here my configuration.yaml entries:

![afbeelding](https://user-images.githubusercontent.com/122681557/216119214-4963d2a4-1259-476f-b34d-59c39d5f4028.png)

**Comment by ekutner** [comment]
This indeed looks fine. Try enabling debug logging (see in the README), restart HA, try adding the integration again and then upload the complete log file.

**Comment by Snipey7** [comment]
Here the log file.

There are some entries in there regarding another (XIaomi) integration which I will have to look into. 

I think these are unrelated as I was struggeling with the HA_alt installation already before I changed something in the Xiami setup which caused the Xiami issues.
[home-assistant_2023-02-04T10-13-11.029Z.log](https://github.com/ekutner/home-connect-hass/files/10608384/home-assistant_2023-02-04T10-13-11.029Z.log)

**Comment by ekutner** [comment]
The log doesn't have any useful information either.
It's a stretch, but since the error you are getting indicates a configuration problem, I can suggest two options. If you are comfortable doing that, you can upload your config for me to review. Alternatively, try to completely delete the home_connect_alt configuration section, then type it again *manually* (not using copy and paste). Make sure the "home_connect_alt" line starts at the beginning of the row. This should guarantee there are now non-printable characters in those lines that may have been accidentally copied from the web page.
Note that you don't have to manually type the client ID and secret, you can just put something random there to begin with because the error you got is before the values are actually used. If after this a popup window opens with an error, then you should put in the correct values for client ID and secret.
Another thing worth trying is removing the integration from HACS, restarting HA then adding it back and restarting again.
These are all the ideas I have.

**Comment by Snipey7** [comment]
I discovered that I made the mistake of copying in the configuration lines in my secrets.yaml. I corrected this and now it works fine.

Thank you for the support.
